data_collection/gazette/spiders/base/portalgov.py

```diff
@@ -0,0 +1,75 @@
+import re
+from datetime import datetime as dt
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class BasePortalGovSpider(BaseGazetteSpider):
+    power = "executive"
+
+    def start_requests(self):
+        domain = self.allowed_domains[0]
+        yield scrapy.FormRequest(
+            url=f"https://{domain}/controllers/diario_oficial/class_diario.php",
+            method="POST",
+            formdata={
+                "func": "5",
+                "param": "1",
+            },
+        )
+
+    def parse(self, response):
+        domain = self.allowed_domains[0]
+        BASE_URL = f"https://{domain}/arquivos/diario_oficial/"
```

Comment:
Deixaria minúsculo por não ser constante
```suggestion
        base_url = f"https://{domain}/arquivos/diario_oficial/"
```

Reply:
Sugestão aplicada.